roles/network-multus/tasks/deprovision.yml

```diff
@@ -25,6 +25,40 @@
 - name: Delete cni plugins Resources
   command: "{{ cluster_command }} delete -f /tmp/cni-plugins.yml --ignore-not-found"
 
+# note: this rule does NOT delete /etc/pcidp/config.json file because k8s
+# doesn't support finalizers to clean up mounted directory
+- name: Uninstall SR-IOV device plugin
+  block:
+  - name: Render SR-IOV DP deployment yaml
+    template:
+      src: sriovdp.yml
+      dest: /tmp/sriovdp.yml
+  - name: Delete SR-IOV DP resources
+    shell: "{{ cluster_command }} delete -f /tmp/sriovdp.yml --ignore-not-found"
+  when: deploy_sriov_plugin
+
+- name: Uninstall SR-IOV CNI plugin
+  block:
+  - name: Render SR-IOV CNI deployment yaml
+    template:
+      src: sriov-cni.yml
+      dest: /tmp/sriov-cni.yml
+  - name: Delete SR-IOV CNI resources
+    shell: "{{ cluster_command }} delete -f /tmp/sriov-cni.yml --ignore-not-found"
+  when: deploy_sriov_plugin
+
+- name: Uninstall SR-IOV network
+  block:
+  - name: Render SR-IOV network CRD yaml
```

Comment:
Just curious: Why do we deploy an SR-IOV network attachment definition ourselves?

Reply:
This is not a concern to address in this PR, but in general, the idea is that VMIs can be attached to SR-IOV right after deployment. The CRD, in case of SR-IOV, is to reflect pool name that is configured in config.json for DP, which is hardcoded right now. I find it logical that if we configure DP then we should also configure multus network to consume the resource. Again, this is a separate discussion from this PR.